src/project-manager.ts

```diff
@@ -238,14 +321,27 @@ export class ProjectManager implements Disposable {
 		return this.ensureAllFiles();
 	}
 
+	/**
+	 * Recursively collects file(s) dependencies up to given level.
+	 * Dependencies are extracted by TS compiler from import and reference statements
+	 *
+	 * Dependencies include:
+	 * - all the configuration files
+	 * - files referenced by the given file
+	 * - files included by the given file
+	 *
+	 * @param fileNames files to process
```

Comment:
Function accepts both relative and absolute paths. Updated documentation, renamed variables and arguments